ibm/provider.go

```diff
@@ -288,6 +288,7 @@ func Provider() *schema.Provider {
 			"ibm_is_volume_profiles":                 dataSourceIBMISVolumeProfiles(),
 			"ibm_is_vpc":                             dataSourceIBMISVPC(),
 			"ibm_is_vpn_gateways":                    dataSourceIBMISVPNGateways(),
+			"ibm_is_vpc_address_prefix":              dataSourceIbmIsVpcAddressPrefix(),
```

Comment:
can you change to ibm_is_vpc_address_prefixs ( because it gives a list of prefixes)